backend/.gitignore

```diff
@@ -1,7 +1,8 @@
-node_modules
-.env
-
-# macOS
-.DS_Store
-
-.vscode/
+node_modules
+.env
+
+# macOS
+.DS_Store
+
+.vscode/
+**/package-lock.json

```

Comment:
adding package-lock.json it's not an eslint error to be fixed here
please create a new issue for putting it into the gitignore files, and leave it in the backlog. 
we'll do it in this sprint if we have time, but now focus on the tasks that are already in this sprint.